draft-ietf-oauth-browser-based-apps.md

```diff
@@ -307,15 +308,21 @@ The Application Server SHOULD be considered a confidential client, and issued it
 In this scenario, the connection between the browser and Application Server SHOULD be a
 session cookie provided by the Application Server.
 
+### Security considerations
 Security of the connection between code running in the browser and this Application Server is
 assumed to utilize browser-level protection mechanisms. Details are out of scope of
 this document, but many recommendations can be found in the OWASP Cheat Sheet series (https://cheatsheetseries.owasp.org/),
 such as setting an HTTP-only and `Secure` cookie to authenticate the session between the
 browser and Application Server.
 
-<!--
-TODO: security considerations around things like Server Side Request Forgery or logging the cookies
+Additionally, cookies MUST be protected from leakage by other means, such as logs.
 
+This architecture protects against tokens leakage from the browser, but creates a CSRF attack vector:
+once the user is authenticated, the BFF proxy will automatically add tokens to calls to the resource server.
+
+Cookies must be protected, not only from the browser itself using
```

Comment:
Looks like an incomplete thought? We can just remove this for now to merge this in if you like.

Reply:
Indeed, removed for now.